HACKING.md

```diff
@@ -180,70 +184,73 @@ reason we create the app first is that otherwise OpenShift will
 automatically instantiate Jenkins with default parameters when creating
 the Jenkins pipeline).
 
-Next, let's update the input ImageStreamTag for our Jenkins deployment
-to match latest OpenShift (`jenkins:2`). Some older versions of the
-template in OpenShift uses `jenkins:latest`. This will no longer be
-needed once we are running on a newer version of OpenShift than 3.6 in
+The `jenkins:2` parameter is to match the tag name in the latest
+OpenShift. Some older versions of the template in OpenShift uses
+`jenkins:latest`. This will no longer be needed once we are running on a
+newer version of OpenShift than 3.6 in
 CentOS CI. See [#32](https://github.com/coreos/fedora-coreos-pipeline/pull/32)
 and [#70](https://github.com/coreos/fedora-coreos-pipeline/pull/70)
-for context on why this may be needed):
+for more context).
+
+### Creating the pipeline
+
+If working on the production pipeline, you may simply do:
 
 ```
-oc patch dc/jenkins -p '{"spec":{"triggers":[{"imageChangeParams":{"automatic":true,"containerNames":["jenkins"],"from":{"kind":"ImageStreamTag","name":"jenkins:2"}},"type":"ImageChange"},{"type":"ConfigChange"}]}}'
+./deploy --official --update
 ```
 
-### Create the pipeline from the template
+If working in a different namespace/cluster, the command is the same,
+but without the `--official` switch:
 
-If working on the production pipeline, you may simply do:
 
 ```
-oc new-app --file=manifests/pipeline.yaml
+./deploy --update
 ```
 
-If working on a local cluster you will want to override the defaults
-for some parameters:
+You may also want to provide additional switches depending on the
+circumstances. Here are some of them:
 
-- `DEVEL_PREFIX` - **REQUIRED**
-    - The prefix to prepend to created resources.
-    - This is recommended to be your username followed by a dash.
-      i.e. `jlebon-`. The pipeline will simply not run without a
-      `DEVEL_PREFIX` set if it is not running in prod mode.
-- `PIPELINE_REPO_URL`
-    - Git source URI for pipeline Jenkinsfile
-- `PIPELINE_REPO_REF`
-    - Git branch/tag reference for pipeline Jenkinsfile
-- `PIPELINE_FCOS_CONFIG_URL`
-    - Git source URI for FCOS config
-- `PIPELINE_FCOS_CONFIG_REF`
-    - Git branch/tag reference for FCOS config
-- `PVC_SIZE`
-    - Size of the PVC to create
-- `COREOS_ASSEMBLER_IMAGE`
-    - Image of coreos-assembler to use
-- `S3_BUCKET`
-    - AWS S3 bucket in which to store builds (or blank for none)
+- `--prefix`
+    - The prefix to prepend to created developer-specific resources. By
+      default, this will be your username, but you can provide a
+      specific prefix as well.
+- `--pipeline`
+    - Git source URI for pipeline Jenkinsfile.
+- `--config`
+    - Git source URI for FCOS config.
```

Comment:
since before we had the `_REF` parameters and now we don't  you might want to mention the `@ref` formatting for these two

Reply:
Fixed!